tests/app/test_mock_case_service.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+
+class CaseServiceTestCase(unittest.TestCase):
+    def test_some_function(self):
+        assert True
```

Comment:
Case service isn't giving us any information we're actually using I think so this is probably fine